src/MapfileStyleParser.ts

```diff
@@ -382,33 +387,48 @@ export class MapfileStyleParser implements StyleParser {
     }
 
     const symbolType = mapfileStyle.symbol.type.toLowerCase();
-    const points = mapfileStyle.symbol.points.split(' ').map((item) => parseFloat(item));
-    if (symbolType === 'ellipse') {
-      if (!(points[0] === points[1] && points.length === 2)) {
-        console.warn('Custom ellipse not supported by MarkerSymbolyzer, fallback to "Circle"!');
-      }
-      markSymbolizer.wellKnownName = 'circle' as WellKnownName;
-    } else {
-      if (isSquare(points)) {
-        markSymbolizer.wellKnownName = 'square' as WellKnownName;
+    if (mapfileStyle.symbol.points) {
+      const points = mapfileStyle.symbol.points.split(' ').map((item) => parseFloat(item));
+      if (symbolType === 'ellipse') {
+        if (!(points[0] === points[1] && points.length === 2)) {
+          console.warn('Custom ellipse not supported by MarkerSymbolyzer, fallback to "Circle"!');
+        }
+        markSymbolizer.wellKnownName = 'circle' as WellKnownName;
+      } else {
+        if (isSquare(points)) {
+          markSymbolizer.wellKnownName = 'square' as WellKnownName;
+        }
+        if (isTriangle(points)) {
+          markSymbolizer.wellKnownName = 'triangle' as WellKnownName;
+        }
+        if (isCross(points)) {
+          markSymbolizer.wellKnownName = 'cross' as WellKnownName;
+        }
       }
-      if (isTriangle(points)) {
-        markSymbolizer.wellKnownName = 'triangle' as WellKnownName;
+      if (!markSymbolizer.wellKnownName) {
+        console.warn(
+          `Custom symbol not supported by MarkerSymbolyzer, fallback to 'X':\n${JSON.stringify(
+            mapfileStyle.symbol,
+            null,
+            2
+          )}`
+        );
+        markSymbolizer.wellKnownName = 'X' as WellKnownName;
```

Comment:
see comment above

Reply:
All WellKnownNames are now lowercase so this should be `'x'`.